### ISSUE TITLE
SA_KILL does not superseeds SPAMASSASSIN_SPAM_TO_INBOX

### ISSUE THREAD

**Issue by frugan-it** [docker-mailserver/docker-mailserver#2491]
# Subject

I would like to configure a not well documented mail server use case

## Description

In the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/faq/#how-do-i-have-more-control-about-what-spamassasin-is-filtering) it says:

> SA_KILL needs to be very high, to make sure nothing is bounced at all (SA_KILL superseeds SPAMASSASSIN_SPAM_TO_INBOX)

but, if `SPAMASSASSIN_SPAM_TO_INBOX=1`, it does not appear to me that *SA_KILL superseeds SPAMASSASSIN_SPAM_TO_INBOX*.

SPAM emails that exceed the `SA_KILL` level are still sent to the inbox, because (always when `SPAMASSASSIN_SPAM_TO_INBOX=1`) these 2 variables are set as `D_PASS`:

**/etc/amavis/conf.d/49-docker-mailserver**:
```
$final_spam_destiny       = D_PASS;    
$final_bad_header_destiny = D_PASS;
```
by `setup-stack.sh` script:

https://github.com/docker-mailserver/docker-mailserver/blob/321ae744fa873ca300af83983b4f916ac8dc9c35/target/scripts/startup/setup-stack.sh#L930-L931

Mind you, this is exactly how I want to work, I don't want SPAM emails to be bounced.
I just wanted to point out that IMO what is indicated in the documentation is not correct..

In a nutshell: it is not necessary to set a high value of `SA_KILL` when `SPAMASSASSIN_SPAM_TO_INBOX=1`.
Correct me if I'm wrong..

**Comment by georglauterbach** [comment]
Thank you for sharing this. Please provide a PR that the maintainers can review if you want the documentation to change :)
